### DISCUSSION TITLE
Display Scaling Factor

### DISCUSSION THREAD

**Discussion by zimbawe998** [elementary/switchboard-plug-display#340] [Q&A]
Is it possible to have a different scaling factor when you have two display?
This is useful when you need to share some contents with customer that haven't a new device or device that allow a specific resolution.

**Comment by raphaelsuzuki** [comment]
Wayland ("secure session" option in the login screen) fixes this, giving us independent scaling factors for each screen, a feature that was only possible in X11 using xrandr. The problem (for me, at least) is the very limited options for scaling: 100%, 200%, 300% and 400%. A "custom scaling factor" option would be optimal, since it is very dependent on the monitor size/resolution. The "fix" I'm using right now is using the text size under "Desktop" configurations, but this option applies to both monitors (making text on one of them look very big) and doesn't seem to have an effect on all applications.
